build.gradle

```diff
@@ -65,3 +65,8 @@ task functional(type: Test) {
   }
 }
 tasks.functional.dependsOn("myZip");
+
+run {
+    args System.getProperty("args", "").split()
```

Comment:
One last comment 
- can you add a comment regarding the second parameter `""`?

Reply:
It represents a default value, but we can actually do without the default value as well. So, I will go ahead and just have the first parameter.